frontend/src/metabase/visualizations/components/Visualization/Visualization-table.unit.spec.js

```diff
@@ -50,10 +50,15 @@ describe("Table", () => {
 
     const bgColors = rows
       .map(([value]) => screen.getByText(String(value)))
-      .map(element => element.parentNode.style["background-color"]);
+      .map(element =>
+        window
+          .getComputedStyle(element.parentNode)
+          .getPropertyValue("background"),
+      );
```

Comment:
Mantine generates Emotion class names under the hood, so our tests must use `getComputedStyle`